data/views/node.2.0.json

```diff
@@ -13,18 +21,24 @@
 <% } else  {%>
 "tags": [],
 <% } %>
-<% if (hasOwnProperty("obmSettings")) { %>
-"obmSettings": <%- JSON.stringify(obmSettings) %>,
-<% } else  {%>
-"obmSettings": [],
+<% if (obms == null) { %>    
+"obms": null,
+<% } else { %>
+"obms":[
+<% obmId.forEach(function (ids, i, arr){ %>
+{
+"obm":"<%=basepath %>/obms/<%=ids%>"
+}
+<%= ( arr.length > 0 && i < arr.length-1 ) ? ',': '' %>
+<% }); %>
+],
 <% } %>
-"type": "<%=type%>",
 <% if (hasOwnProperty("sku")) { %>
 <% if (sku === null) { %>
```

Comment:
Also for this line, if the `sku` is not existing in the rendering context, this will fail.